README.md

```diff
@@ -1,79 +1,82 @@
-# DroneKit LA
+# DroneKit LogAnalyzer (LA)
```

Comment:
I've been writing this the opposite way and with a space between Log Analyzer **DroneKit LA (Log Analyzer)**, I think it reads better and it differentiates itself further from the current **LogAnalyzer** in Ardupilot